src/caffe/layers/cudnn_conv_layer.cpp

```diff
@@ -51,7 +51,7 @@ void CuDNNConvolutionLayer<Dtype>::LayerSetUp(
       this->num_output_ / this->group_, this->channels_ / this->group_,
       kernel_h, kernel_w);
 
-  this->weight_offset_ = (this->num_output_ / this->group_) *
```

Comment:
weight_offset_ is used in 3 places in cudnn_conv_layer.cu
(e.g., https://github.com/drnikolaev/caffe/blob/caffe-0.15-workspace-fix/src/caffe/layers/cudnn_conv_layer.cu#L39)

Now that it is not initialized in Setup, does it get the right value?